community/kernel/src/main/java/org/neo4j/helpers/collection/Visitable.java

```diff
@@ -19,7 +19,7 @@
  */
 package org.neo4j.helpers.collection;
 
-public interface Visitable<T>
+public interface Visitable<V>
 {
-    void accept(Visitor<T, RuntimeException> visitor);
+    void accept( V visitor );
```

Comment:
Don't understand the reason to change this interface. I think you need an adhoc one for your use case or make an adapter between the `Visitor` interface and your specific one.